src/nvme/types.h

```diff
@@ -2077,6 +2077,10 @@ struct nvme_zns_lbafe {
  * @mor: Maximum Open Resources
  * @rrl: Reset Recommended Limit
  * @frl: Finish Recommended Limit
+ * @numzrwa: Number of ZRWA Resources
+ * @zrwafg: ZRWA Flush Granularity
+ * @zrwasz: ZRWA Size
+ * @zrwacap: ZRWA Capability
```

Comment:
Please also align these description. While at it, you could also align the rest of the documentation :)